README.md

```diff
@@ -2,10 +2,9 @@
 
 ```shell
 pip install pre-commit detect-secrets
-mkdir .gds
-detect-secrets scan > .gds/.secrets.baseline
-detect-secrets audit .gds/.secrets.baseline
-pre-commit install -c .gds/.pre-commit-config.yaml #this file is from this repo
+mkdir ~/gds
```

Comment:
What happens if the user already has a folder called ~/gds? I would have thought that's quite likely. Maybe we should call it ~/gds-pre-commit ~/gds/github-pre-commit or something that's not likely to be used for something else. If you're creating folder structure within that then maybe it's just a mkdir -p so it doesn't fail if it's already there.

Reply:
yeah, we're gonna parameterize this

Reply:
Actually, we've moved do `.gds-pre-commit/`